app/views/notifier/order_receipt.html.haml

```diff
@@ -31,19 +32,23 @@
         %td= od
         - if od.reservation
           %td
-            = ((od.bundle ? "#{h od.bundle} &mdash; " : '') + h(od.product)).html_safe
+            = ((od.bundle ? "#{od.bundle} &mdash; " : "") + h(od.product)).html_safe
```

Comment:
This is a confusing one. Looking again, I think what it needs to be is:
`(od.bundle ? "#{h od.bundle} &mdash; ".html_safe : od.product)`

I'll update and test later when I'm off the train.